streaming/src/main/scala/org/apache/gearpump/streaming/task/Subscription.scala

```diff
@@ -61,9 +61,9 @@ class Subscription(
   private val processingWatermarkSince: Array[Short] = new Array[Short](parallelism)
 
   private val outputWatermark: Array[MilliSeconds] = Array.fill(parallelism)(
-    Watermark.MIN.toEpochMilli)
+    Watermark.MAX.toEpochMilli)
```

Comment:
This doesn't look right since the `watermark` method will return maximum value when the outgoing messages are not acked yet